src/main/webapp/payment.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Оплата товара</title>
+</head>
+<body>
+<h1>Создание заказа</h1>
+<form action="/payment" method="post">
+    E-mail <input type="text" name="email" value="${userEmail}"><br>
+    Номер телефона <input type="tel" value="+123456789012"><br>
```

Comment:
where we work with this data?
better create separate entity - Order and work with them